src/VscodeContext.ts

```diff
@@ -0,0 +1,26 @@
+import * as vscode from 'vscode';
+
+type ContextValue = boolean | string;
+
+/**
+ * Wrapper around VS Code's `setContext`.
+ * The API call takes several milliseconds to seconds to complete,
+ * so let's cache the values and only call the API when necessary.
+ */
+// eslint-disable-next-line @typescript-eslint/no-extraneous-class
+export abstract class VSCodeContext {
```

Comment:
Our rule disallows this. I brought this example from [vscodeContext}(https://github.com/VSCodeVim/Vim/blob/a4656a6e53a81e1ae4e2406f81ad831afffd0a18/src/util/vscodeContext.ts) in the vim extension. Found it to be a clean and easy way to mange custom contexts.
Let me know if we should change this.

Reply:
Fixed this in a recent commit